grouper/fe/handlers.py

```diff
@@ -717,6 +718,136 @@ def get(self, group_id=None, name=None):
         )
 
 
+class AuditsComplete(GrouperHandler):
```

Comment:
I'm not generally a fan of the overloading of "auditing", but if it's standard parlance I can learn to live with it.  To me, "an audit" should refer unambiguously to the top-level concept that applies when auditors visit an institution.  As such, I'm in favor of replacing "group audit" with a synonym like "probe" or "review", depending on how scary/invasive we want it to sound.  It's not so obvious from its name whether this model is going to refer to a view of past top-level audits, the status of current group audits, or something else.